app/src/main/res/values-zh-rCN/strings.xml

```diff
@@ -161,13 +161,13 @@
     <!-- Label indicating that sync is in progress -->
     <string name="sync_syncing_in_progress">正在同步…</string>
     <!-- Label summary indicating that sync failed. The first parameter is the date stamp showing last time it succeeded -->
-    <string name="sync_failed_summary">同步失败。最后同步：%s</string>
+    <string name="sync_failed_summary">同步失败。上次成功：％s</string>
```

Comment:
This change looks like it's using the wrong `%` now? @Pike?

Reply:
Good catch, let me fix that upstream and update the PR.

Reply:
FYI, just checked, this is equivalent to a missing placeable. compare-locales does check for params not in en-US, and for mismatching params. It doesn't check for params to be left out, as that's sometimes OK, for example when en-US refers to branding. Some languages always use passive voice there, so the brand name doesn't show up at all.